CalculatorKata/Program.cs

```diff
@@ -12,8 +12,8 @@ public int add(String numbers)
         {
             if (string.IsNullOrWhiteSpace(numbers))
                 return 0;
-            char[] delimiterChars = { '\n', ',' };
-            string[] num = numbers.Split(delimiterChars);
+            char[] delimiterChars = { '\n', ',', ';', '/' };
```

Comment:
if I wanted to use the '$' as a delimiter, will this allow me to?
The requirement is to allow for any delimiter